sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -253,9 +253,8 @@ object SharedState extends Logging {
     val warehousePath = if (hiveWarehouseDir != null && sparkWarehouseOption.isEmpty) {
       // If hive.metastore.warehouse.dir is set and spark.sql.warehouse.dir is not set,
       // we will respect the value of hive.metastore.warehouse.dir.
-      sparkConf.set(WAREHOUSE_PATH.key, hiveWarehouseDir)
       logInfo(s"${WAREHOUSE_PATH.key} is not set, but $hiveWarehouseKey is set. Setting" +
```

Comment:
We still need this `logInfo` even after the code in L256 is removed?

Reply:
These log messages look not very useful to me personally before or after this change. But we do not vary the logging logic with this change, I am not sure if there is anyone who relies on these outputs